client/packages/common/src/intl/locales/en/common.json

```
@@ -2043,6 +2044,7 @@
   "preference.authoriseGoodsReceived": "Authorise goods received",
   "preference.authorisePurchaseOrder": "Authorise purchase orders",
   "preference.customTranslations": "Custom translations",
+  "preference.disableManualReturns": "Disable Manual Returns",
```

Nit: sentence casing pls~

```suggestion
  "preference.disableManualReturns": "Disable manual returns",
```